pkgs/servers/unifi/default.nix

```diff
@@ -52,7 +52,7 @@ in rec {
   };
 
   unifi6 = generic {
-    version = "6.4.54";
```

Comment:
The old version here doesn't match the version in the commit message. Is that intentional?

Reply:
no, forgot to change the commit message.